xml/System.Net/WebHeaderCollection.xml

```diff
@@ -1448,49 +1448,7 @@
         <exception cref="T:System.ArgumentOutOfRangeException">The length of <paramref name="value" /> is greater than 65535.</exception>
         <exception cref="T:System.InvalidOperationException">This <see cref="T:System.Net.WebHeaderCollection" /> instance does not allow instances of <see cref="T:System.Net.HttpResponseHeader" />.</exception>
       </Docs>
-    </Member>
```

Comment:
@gewarren I decided to remove `Item[string]`completely as to be consistent with `Item[int]`. Both properties are on the base class. 
Do you agree?

Reply:
Hmm, this will just get re-added the next time we do a CI build to import API changes. I'm wondering why it doesn't show "Inherited from NameValueCollection" on this page:

![image](https://user-images.githubusercontent.com/24882762/84341015-f853cc80-ab56-11ea-8f3b-295533bc4344.png)

@mairaw Have you seen this before? Seems like maybe a bug in mdoc or something?

Reply:
`Item[Int32]` and `Item[String]` are the same cases, however `Item[Int32]` is not part of `WebHeaderCollection.xml` but `Item[String]` is. As missing `Item[Int32]` is correctly documented I expect it is correct that it is missing in the xml doc. The question is why one property was generated while the other not? There is a link to the WebHeaderCollection source code for a reference: [WebHeaderCollection.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.WebHeaderCollection/src/System/Net/WebHeaderCollection.cs)

Reply:
I'm [asking the docs engineering team](https://teams.microsoft.com/l/message/19:8c58aebc43fd4b80ae92c1fa2f985652@thread.skype/1591893757844?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=8b1528fc-983a-4b43-add2-79fac8efa1e1&parentMessageId=1591893757844&teamName=Developer%20Relations%20Web%20Team&channelName=FC%20-%20.NET%20Infrastructure&createdTime=1591893757844) about this.

Reply:
It's a bug in the tool that creates the XML files, so you can remove it and hopefully by the next release, the bug will be fixed and we won't re-import it.

https://ceapex.visualstudio.com/Engineering/_workitems/edit/235686